Resources/Private/Templates/FrontEndEditor/Index.html

```diff
@@ -40,6 +40,12 @@ <h2>
                                     <f:translate key="plugin.frontEndEditor.action.edit"/>
                                 </f:link.action>
                             </td>
+                            <td>
+                                <f:form action="delete" name="tea" object="{tea}">
```

Comment:
I can't remember when I did this the last time. So to be sure: Extbase/Fluid will add a hidden input with the uid of the record only by adding the object to the form? Or do we need to add that input field ourself?

Reply:
Exactly. In 11LTS, there 7 hidden fields get auto-added. One of them is for the identity:

```php
<input type="hidden" name="tx_tea_teafrontendeditor[tea][__identity]" value="2">
```